src/Symfony/Component/OptionsResolver/OptionConfigurator.php

```diff
@@ -0,0 +1,130 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OptionsResolver;
+
+use Symfony\Component\OptionsResolver\Exception\AccessException;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+
+final class OptionConfigurator
+{
+    /**
+     * The name of the defined option
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var OptionsResolver
+     */
```

Comment:
These phpdoc blocks can be removed as the properties typehint can be guessed from the `__construct` typehints.